CHANGES.md

```diff
@@ -1,3 +1,55 @@
+# v2.1.0
+
+## Major changes
+- Add Python interface
+- Adopting the Frictionless Data specification standards
```

Comment:
Can you clarify the distinction between this and the change in 2.0. Does this mean "Use Frictionless Data specification terminology for internals" or something like that. If so that might be a minor change.